### ISSUE TITLE
'str' object has no attribute 'iteritems'

### ISSUE THREAD

**Issue by cawilliamson** [mtneug/ansible-modules-plist#2]
Hey there!

This is possibly just me using the module wrong here but I am having the following error when I try and use anything other than a single value:

`'str' object has no attribute 'iteritems'`

A chunk of code which produces this in Ansible is:

```
- name: sort finder icons by name
  plist_file:
    dest: com.apple.finder
    value:
      'StandardViewSettings':
        'IconViewSettings':
          'arrangeBy': 'Name'
```

Any ideas here? I'm thinking somehow the value input is being interpreted by the python script as a single string instead of ending up as a dict.

Thanks!

**Comment by mtneug** [comment]
Hi, thank you for reporting. I could reproduce this error. The last time I used this module is already some time ago. This seems to be a [change in Ansible 2.1](https://github.com/ansible/ansible/issues/16057). If I find time I will look into it deeper.

**Comment by mtneug** [comment]
Closed with #3
